include/dbstructure.php

```diff
@@ -1092,39 +1092,39 @@ function db_definition($charset) {
 			);
 	$database["oembed"] = array(
 			"fields" => array(
-					"url" => array("type" => "varchar(255)", "not null" => "1", "primary" => "1"),
+					"url" => array("type" => "varbinary(255)", "not null" => "1", "primary" => "1"),
```

Comment:
I should change that ;-) 

BTW: We had to check the behaviour concerning IDN. Had someone checked if curl is supporting that at all?

But: Even if we had problems in these two tables, it wouldn't matter much, since it only affects cached data.